ArduPlane/ArduPlane.cpp

```diff
@@ -415,6 +415,8 @@ void Plane::update_GPS_10Hz(void)
     }
 
     calc_gndspeed_undershoot();
+	Estimate_Bearing_SPD_ETA();
```

Comment:
Indenting - lots of whitespace fixes need in here.  Please conform to the code around your new stuff - and check our style guide for what we like to see.